src/libslic3r/GCode/GCodeProcessor.cpp

```diff
@@ -59,8 +59,14 @@ const std::vector<std::string> GCodeProcessor::Reserved_Tags = {
     "_GP_LAST_LINE_M73_PLACEHOLDER",
     "_GP_ESTIMATED_PRINTING_TIME_PLACEHOLDER",
     "_GP_TOTAL_LAYER_NUMBER_PLACEHOLDER",
+    "_GP_TOTAL_FILAMENT_USED_MM_PLACEHOLDER",
+    "_GP_TOTAL_FILAMENT_USED_CM3_PLACEHOLDER",
+    "_GP_TOTAL_FILAMENT_USED_G_PLACEHOLDER",
```

Comment:
Same

Reply:
> Same

@XunZhangBambu Would you like me to remove all of these entries from 59-64?